src/mlpack/methods/ann/not_adapted/brnn_impl.hpp

```diff
@@ -384,10 +384,10 @@ EvaluateWithGradient(const arma::mat& /* parameters */,
 
   if (backwardGradient.is_empty())
   {
-    backwardGradient = arma::zeros<arma::mat>(
+    backwardGradient = zeros<arma::mat>(
         parameter.n_rows/ 2,
         parameter.n_cols);
-    forwardGradient = arma::zeros<arma::mat>(
+    forwardGradient = zeros<arma::mat>(
         parameter.n_rows/ 2,
         parameter.n_cols);
```

Comment:
Both of these (`backwardGradient` and `forwardGradient`) can fit on one line now. :+1: